backend/src/main/kotlin/no/uib/echo/routes/UserRoutes.kt

```diff
@@ -315,3 +318,62 @@ fun Route.getToken(env: Environment, audience: String, issuer: String, secret: S
         call.respond(token)
     }
 }
+
+fun Route.getWhitelist() {
+    get("/user/whitelist") {
+        val email =
+            call.principal<JWTPrincipal>()?.payload?.getClaim("email")?.asString()?.lowercase()
+
```

Comment:
Bør også ha denne bak autorisasjon

```suggestion
        if (email !in getGroupMembers("webkom")) {
            call.respond(HttpStatusCode.Unauthorized)
            return@put
        }
```

Reply:
Den er jo det? 

```kotlin
authenticate(jwtConfig) {
          postUser()
          putUser()
          getAllUsers()
          getWhitelist()
          putWhitelist()
      }
```

Reply:
Åja nvm. Ja kan bruke `ADMIN_KEY`.

Reply:
Eller man kan jo bare sjekke seg selv slik det er nå, siden email kommer fra token.

Reply:
åja trodde det var å hente ut alle på whitelist